Week1/Js-exercises/6-array-of-animals.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+let animalsArr;
+console.log("I think the value will be undefined");
+console.log(animalsArr);
+let favoriteAnimals = ["Hamsters", "Cats", "Dogs",];
+console.log(favoriteAnimals);
+favoriteAnimals.push("Horses");
+console.log(favoriteAnimals);
```

Comment:
Could use `const` instead of `let` here 🙂

Reply:
Fixed